operations/patch_util.go

```diff
@@ -676,6 +681,24 @@ func loadGitData(dir, branch, ref, commits string, format bool, extraArgs ...str
 	}, nil
 }
 
+func gitGetRemote(dir, owner, repo string) (string, error) {
+	out, err := gitCmdWithDir("remote", dir, "-v")
+	if err != nil {
+		return "", errors.Wrap(err, "getting git remotes")
+	}
+
+	lines := strings.Split(out, "\n")
+	partial := strings.ToLower("github.com/" + owner + "/" + repo)
```

Comment:
nit: might be more readable to use `fmt.Sprintf("github.com/%s/%s", owner, repo)` rather than concatenate.